source/extensions/filters/network/kafka/protocol_code_generator/complex_type_template.j2

```diff
@@ -0,0 +1,37 @@
+struct {{ complex_type.name }} {
```

Comment:
Can we have some comments at the top of each template file explaining roughly what the template is for? It can be hard to dive into the template code otherwise.